CHANGELOG.md

```diff
@@ -10,7 +10,9 @@
 
 ##### Bug Fixes
 
-* None.
+* Fix some tests fail with Swift 5.0 on macOS.  
```

Comment:
Fix some test failures with the latest Swift 5 snapshot on Apple platforms.

Reply:
Thanks! 🙏